app/Livewire/Project/Service/Storage.php

```
@@ -39,30 +67,151 @@ public function refreshStoragesFromEvent()
+            $this->authorize('update', $this->resource);
+
+            $this->validate([
+                'file_storage_path' => 'required|string',
+                'file_storage_content' => 'nullable|string',
+            ]);
+
+            $this->file_storage_path = trim($this->file_storage_path);
+            $this->file_storage_path = str($this->file_storage_path)->start('/')->value();
+
+            if ($this->resource->getMorphClass() === \App\Models\Application::class) {
+                $fs_path = application_configuration_dir().'/'.$this->resource->uuid.$this->file_storage_path;
```

PHPMD (2.15.0)

The variable $fs_path is not named in camelCase. (undefined)

(CamelCaseVariableName)